sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Azure.Messaging.EventHubs.Processor.Tests.csproj

```diff
@@ -33,4 +33,8 @@
 
   <!-- Import the Azure.Core test framework (shared source) -->
   <Import Project="..\..\..\core\Azure.Core\tests\TestFramework.props" />
+  <!-- Import the shared infrastructure from the Event Hubs client tests (shared source) -->
```

Comment:
This should no longer be a thing; all of the shared code is coming from the `Shared` project and should already be referenced.